test-requirements.txt

```diff
@@ -10,4 +10,6 @@ recommonmark
 codecov>=1.4.0
 pycodestyle
 autopep8
-isort
\ No newline at end of file
+isort
+jwt
+adal
```

Comment:
why do we change test requirements?

Reply:
I don't think we're using jwt or adal?

Reply:
@micw523, the test failed with import error without jwt and adal.

Reply:
I checked again:
```
$ grep -nr adal kubernetes | grep -v \.pyc
kubernetes/base/config/kube_config.py:24:import adal
kubernetes/base/config/kube_config.py:223:        context = adal.AuthenticationContext(
```

jwt is a dependency of adal, and I'm not sure adal is specifically required here.
I removed jwt. How do you want me to proceed with adal?